docs/hub/model-cards.md

```diff
@@ -89,8 +117,21 @@ This metadata will be used to display the base model on the model page. Users ca
 <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/base-model-ui-dark.png"/>
 </div>
 
+### Specifying a dataset
```

Comment:
I would link this from https://moon-ci-docs.huggingface.co/docs/hub/pr_1116/en/models-faq#how-can-i-see-what-dataset-was-used-to-train-the-model